activerecord/CHANGELOG.md

```diff
@@ -856,6 +856,7 @@
       ```ruby
       class User < ActiveRecord::Base
         normalizes :email, with: -> email { email.strip.downcase }
+        normalizes :phone, with: -> phone { phone.delete("^0-9").delete_prefix("1") }
```

Comment:
@Earlopain This example is a little confusing, how about just `phone.strip`?

Reply:
D'oh, sorry I was mixing up the previous example because you added the `where` below 🤦 sorry!